learn/guides/distinctive-language-features/advanced-general-purpose-language-features.md

```diff
@@ -294,18 +300,18 @@ type Bar object {
 
 type FooBar Foo & Bar;
 
-// same as
+// Same as
 type FooBar object {
     *Foo;
     *Bar;
 };
 ```
 
-In the above code example, **``Foo``** and **``Bar``** are two separate object types. The type **``FooBar``** is defined through the type intersection of **``Foo``** and **``Bar``** using `‘&’` notation. This has the same result as including the types using the `*` notation. It provides a neat alternative to type inclusion, but is less flexible.
+In the above code example, **``Foo``** and **``Bar``** are two separate object types. The **``FooBar``** type is defined using the type intersection of **``Foo``** and **``Bar``** using the ``&`` notation. This has the same result as including the types using the ``*`` notation. It provides a neat alternative to type inclusion but is less flexible.
 
-## Expression-oriented Style
+## Expression-Oriented Style
 
-Ballerina’s general philosophy is to make the programming syntax familiar to those familiar with the C-family of languages. It has a distinction between statements and expressions, just like in C. This is the imperative style of programming which is considered less intuitive. That’s why there is a shift towards a functional style of programming which is more expression oriented..  
+Ballerina's general philosophy is to make the programming syntax familiar to those familiar with the C-family of languages. It has a distinction between statements and expressions, just like in C. This is the imperative style of programming which is considered less intuitive. That's why there is a shift towards a functional style of programming that is more expression-oriented.
```

Comment:
```suggestion
Ballerina's general philosophy is to make the programming syntax familiar to those familiar with the C-family of languages. This is the imperative style of programming which is considered less intuitive. It has a distinction between statements and expressions, just like in C. That's why there is a shift towards a functional programming style that is more expression-oriented.
```

Reply:
Is switching the sentences correct here?

Reply:
Lets keep

Reply:
Lets keep

Reply:
Lets keep